openshift_tools/monitoring/metric_sender.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+# vim: expandtab:tabstop=4:shiftwidth=4
+"""
+Wrapper for concrete Metric Sender classes: ZaggSender, HawkSender etc.
+Check configuration and send metrics using specific client if enabled.
+
+Examples:
+
+    from openshift_tools.monitoring.hawk_common import HawkConnection, HawkHeartbeat
+    from openshift_tools.monitoring.hawk_sender import HawkSender
+    HOSTNAME = 'host.example.com'
+
+    ZAGGCONN = HawkConnection(url='https://172.17.0.151', user='admin', password='pass')
+    ZAGGHEARTBEAT = HawkHeartbeat(templates=['template1', 'template2'], hostgroups=['hostgroup1', 'hostgroup2'])
+
+    zs = HawkSender(host=HOSTNAME, hawk_connection=ZAGGCONN)
+    zs.add_heartbeat(ZAGGHEARTBEAT)
+    zs.add_zabbix_keys({ 'test.key' : '1' })
+    zs.send_metrics()
+"""
+
+from openshift_tools.monitoring.generic_metric_sender import GenericMetricSender
+from openshift_tools.monitoring.zagg_sender import ZaggSender
+from openshift_tools.monitoring.hawk_sender import HawkSender
+
+
+class MetricSender(GenericMetricSender):
+    """
+    collect and create UniqueMetrics and send them to specific senders
+    """
+
+    def __init__(self, host=None, verbose=False, debug=False):
+        '''
+        Read config file and add specific active senders to sender list
+        '''
+        super.__init__
+
+        self.config_file = '/etc/openshift_tools/metric_sender.yaml'
+        self.verbose = verbose
+        self.debug = debug
+        self.active_senders = None
+
+        super.parse_config()
+        if self.is_zagg_active:
```

Comment:
I could be wrong about this, but what's happening here is that since python treats functions as first class citizens, you're checking if the the function exists.

Here's an example:
```python
>>> def blah():
...    print "blah"
... 
>>> blah
<function blah at 0x7ff99e3da758>
>>> blah()
blah
>>> 
```

Note that without the `()` then we get back the function pointer.

I'm pretty sure we want `()` on these if statements.